test/linter/test-consistency.js

```diff
@@ -376,9 +376,6 @@ export class ConsistencyChecker {
       typeof a_version_added === 'string' &&
       typeof b_version_added === 'string'
     ) {
-      if (b_version_added.startsWith('≤')) {
-        return false;
-      }
```

Comment:
We may want to keep this, given removing it leads to existing lint failures (see CI).